front/app/translations/admin/en.json

```diff
@@ -1649,7 +1649,7 @@
   "app.containers.AdminPage.groups.permissions.selectValueError": "Please select an answer type",
   "app.containers.AdminPage.groups.permissions.useExistingRegistrationQuestions": "Ask platform level registration questions",
   "app.containers.AdminPage.groups.permissions.useExistingRegistrationQuestionsDescription": "Use this setting if collecting name, last name, and answers to platform level demographic questions is important to you. It may increase the barrier for participation.",
-  "app.containers.AdminPage.groups.permissions.userFieldsSelectionDescription": "You can ask the existing registration question(s) or create new ones. If a user hasn’t answered to the question(s), they will be asked to answer it before taking the action. Answers to these questions will be stored in user profiles, and will be only accessible to admins.",
+  "app.containers.AdminPage.groups.permissions.userFieldsSelectionDescription": "Responses to demographic questions will get stored in user profiles. Answers to these questions will be accessible to both admins and moderators to easy participation analysis. Questions will only be asked to users who have not answered them before.",
```

Comment:
This was a copy change I missed in a previous change.

Reply:
Since the key isn't being changed (app.containers.AdminPage.groups.permissions.userFieldsSelectionDescription) I don't believe this will trigger a re-translation when merged into the i2 branch.

I think we need to change the last part of the key to: "userFieldsSelectionDescription2" instead perhaps? :)

Reply:
Yes, I'd forgotten that anything on i2 will have already been translated